appserver/admingui/payara-console-extras/src/main/java/fish/payara/admingui/extras/rest/PayaraRestApiHandlers.java

```diff
@@ -587,21 +589,29 @@ public static void updateNotifiers(HandlerContext handlerCtx) {
             } else if (endpoint.contains("monitoring-service-configuration")){
                 restEndpoint = endpoint + "/monitoring-" + name + "-notifier-configure";
                 forMonitoring = true;
+            } else if (endpoint.contains("admin-audit-configuration")) {
+                restEndpoint = endpoint + "/set-admin-audit-service-notifier-configuration";
+                attributes.put("notifier", name);
+                forAdminAudit = true;
             } else {
                 //Unknown service being configured
                 throw new UnknownConfigurationException();
             }
             
-            HashMap<String, Object> attrs = new HashMap<>();
+            
             if (enabledNotifiers.contains(notifier)){
-                attrs.put("enabled", "true");                
+                attributes.put("enabled", "true");                
```

Comment:
Save yourself a lot of code with:
`attributes.put("enabled", enabledNotifiers.contains(notifier)?"true","false")`

Reply:
I don't like ternaries, I think it makes the code less understandable

Reply:
```
attributes.put("enabled", Boolean.toString(enabledNotifiers.contains(notifier)));
```